### ISSUE TITLE
Override SCSS variables

### ISSUE THREAD

**Issue by midzer** [rqrauhvmra/Tobi#23]
Hi all,
somehow I can't manage to override our SCSS variables like `slideMaxHeight` and `slideMaxWidth`.

Until now I tried to override by simply putting
```
$slideMaxHeight: 90vh;
$slideMaxWidth: 90vw;
```
before `@import` statement.
But this has now effect at all: `max-width` and `max-height` stay at original values. Some1 has a clue what I'm doing wrong here?

Happy weekend
midzer

**Comment by rqrauhvmra** [comment]
Hi @midzer 

I commited a few Sass improvements (Import, Variables, Syntax & formatting) in the codeSplittingES6Rewirte brach.

I've [set the variables to be as a default](https://github.com/rqrauhvmra/Tobi/blob/codeSplittingES6Rewrite/src/scss/_variables.scss) in order to overwrite them ahead of time.

```
$slideMaxHeight: 90vh;

$slideMaxHeight: 85vh; !default; // 85vh is only used if $slideMaxHeight is not already set

img {
  max-height: $slideMaxHeight; // height is 90vh
}
```

Unfortunately this is missing in the current master branch.

Best regards
Benjamin

**Comment by midzer** [comment]
To bad I can't get `codeSplittingES6Rewrite` branch to work in my setup:
when I `import Tobi from 'rqrauhvmra__tobi'` webpack throws an error. `import { Tobi } ...` doesn't work either.

**Comment by rqrauhvmra** [comment]
Hi @midzer 

Unfortunately I'm not familiar with webpack but I'll fix this in version 1.8.1 shortly and work diligently on the new one.

Best regards
Benjamin
